mm2src/mm2_main/tests/mm2_tests/best_orders_tests.rs

```diff
@@ -615,7 +615,15 @@ fn zhtlc_best_orders() {
     let rmd = rmd160_from_passphrase(&bob_passphrase);
     let bob_zombie_cache_path = mm_bob.folder.join("DB").join(hex::encode(rmd)).join("ZOMBIE_CACHE.db");
     log!("bob_zombie_cache_path {}", bob_zombie_cache_path.display());
-    std::fs::copy("./mm2src/coins/for_tests/ZOMBIE_CACHE.db", bob_zombie_cache_path).unwrap();
+    let project_root = {
+        let mut current_dir = std::env::current_dir().unwrap();
+        current_dir.pop();
+        current_dir.pop();
+        current_dir
```

Comment:
could u write in comments where such current_dir() is.
like:
```rust
// this should be: DB_ROOT/x/y/zcash_db/etc..
let mut current_dir = std::env::current_dir().unwrap();
// this is: DB_ROOT/x/y/zcash_db
current_dir.pop();
...
```

also doesn't this depend on where from these tests are ran?
like, running the tests from the root of the `komodo-defi-framework` will result in a different current dir than when running from `komodo-defi-framework/mm2src/mm2_main` or `mm2_tests`? which would break the `.join` below?

Reply:
seems like a failed merge conflict